index.js

```diff
@@ -1,14 +1,24 @@
 'use latest'
 const request = require('request')
 const uri = (type) =>  (concept) => `http://webconcepts.info/concepts/${type}/${concept}.json`
+const allConcepts = 'http://webconcepts.info/concepts.json'
 
-var concepts = {
-  'status': uri('http-status-code'),
-  'method': uri('http-method'),
-  'relation': uri('link-relation')
-}
+var concepts = {};
+var conceptList = [];
 
-var conceptList = Object.keys(concepts).join(', ')
+request(allConcepts, (error, response, body) => {
+  if (error || response.statusCode >= 500){
+    return;
```

Comment:
I'm not sure what can be done about a failed initialization. Maybe it should be retried whenever the `/web-concept` command is used if not initialized properly at start?